manifests/Tencent/QQ/0.0.0.0.yaml

```diff
@@ -0,0 +1,19 @@
+Id: Tencent.QQ
+Name: QQ
+Version: 0.0.0.0
```

Comment:
I don't think it's a good idea to leave out the version.

```diff
- Version: 0.0.0.0
+ Version: 9.3.2.26869
```

Reply:
![image](https://user-images.githubusercontent.com/30528433/82525396-d7a1e380-9b63-11ea-965f-14ce73af6b91.png)

I thought 'universal installers' shouldn't specify its version, as in https://github.com/microsoft/winget-pkgs/pull/60; however the original author did changed the version to a proper value so I did the same just now  https://github.com/microsoft/winget-pkgs/pull/530/commits/124388d37d36dd2768bee899dd15b921a0425971